agent/util-scripts/pbench-collect-sysinfo

```diff
@@ -16,22 +16,23 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 # Defaults
 group=default
 dir="/tmp"
-
-# always notify the user, as collection can sometimes take a while
-echo "Collecting system information"
+sysinfo_opts=( all default block libvirt kernel_config sos topology )
```

Comment:
So what I was hoping to see was:
```
sysinfo_opts_default=( block libvirt kernel_config sos topology )
sysinfo_opts=${sysinfo_opts_default[*]}
```

Then as we add other options for collecting information, they would be added to all, but not the default list.